### DISCUSSION TITLE
How to make text bold/italics/underline/highlight/hyperlink in CTkTextbox widget?

### DISCUSSION THREAD

**Discussion by Zingzy** [TomSchimansky/CustomTkinter#1895] [Q&A] [answered]
I want to make selected text bold/italics/underline/highlight/hyperlink in the CTkTextBox Widget... Is that even possible?? If so please please guide me on how to do so 🙏🏻🙏🏻

**Comment by zSynctic** [comment, marked as answer]
Use custom fonts you can search for them, then you can implement it like this:

```
customtkinter.CTkTextbox(self, font=("{name of the font", {size for example -16})
```
